scripts/calibnet_health_check.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+
+set -e
+
+FOREST_PATH="forest"
+FOREST_CLI_PATH="forest-cli"
+
+TMP_DIR=$(mktemp --directory)
+SNAPSHOT_DIRECTORY=$TMP_DIR/snapshots
+LOG_DIRECTORY=$TMP_DIR/logs
+
+usage() {
+  echo "Usage: $0 <PRELOADED_WALLET_STRING>"
+  exit 1
+}
+
+if [ -z "$1" ]
+  then
+    usage
+fi
+
+function cleanup {
+  # Removing temporary directory $TMP_DIR
+  rm -rf "$TMP_DIR"
+
+  $FOREST_CLI_PATH shutdown --force
+
+  sleep 10s
```

Comment:
It's a bit brittle and potentially too long. How about a loop with a timeout, checking every second up to a minute?